src/toil_scripts/batch_alignment/README.md

```diff
@@ -1,47 +1,146 @@
+- `pip install toil-scripts`
+
+If Toil is already installed globally (true for CGCloud users), or there are global dependencies (like Mesos),
+use virtualenv's `--system-site-packages` flag.
+ 
 ## Dependencies
 
 This pipeline has been tested on Ubuntu 14.04, but should also run on other unix based systems.  `apt-get` and `pip`
-often require `sudo` privilege, so if the below commands fail, try prepending `sudo`.  If you do not have sudo 
-privileges you will need to build these tools from source, or bug a sysadmin (they don't mind). 
+often require `sudo` privilege, so if the below commands fail, try prepending `sudo`.  If you do not have `sudo` 
+privileges you will need to build these tools from source, or bug a sysadmin about how to get them (they don't mind). 
 
 #### General Dependencies
 
     1. Python 2.7
-    2. Curl     apt-get install curl
-    3. Docker   http://docs.docker.com/engine/installation/
-    
+    2. Curl         apt-get install curl
+    3. Docker       http://docs.docker.com/engine/installation/
+
 #### Python Dependencies
 
-    1. Toil     pip install toil
-    2. S3AM     pip install --pre s3am (optional, for upload of BAMFILE to S3)
+    1. Toil         pip install toil
+    2. S3AM         pip install --pre s3am (optional, needed for uploading output to S3)
+
+## Inputs
+
+The BWA pipeline requires input files in order to run. The only required input, aside from the sample(s), is a 
+reference genome.  The pipeline can be sped up by specifying URLs for the reference index files, which are generated 
+with `bwa index` and `samtools faidx`.
+
+## General Usage
  
-## Output
+1. Type `toil-bwa generate` to create an editable manifest and config in the current working directory.
+2. Parameterize the pipeline by editing the config.
+3. Fill in the manifest with information pertaining to your samples.
+4. Type `toil-bwa run [jobStore]` to execute the pipeline.
+
+## Example Commands
+
+Run sample(s) locally using the manifest
+1. `toil-bwa generate`
+2. Fill in config and manifest
+3. `toil-bwa run ./example-jobstore`
+
+Toil options can be appended to `toil-bwa run`, for example:
+`toil-bwa run ./example-jobstore --retryCount=1 --workDir=/data`
+
+For a complete list of Toil options, just type `toil-bwa run -h`
+
+Run a variety of samples locally
+1. `toil-bwa generate-config`
+2. Fill in config
+3. `toil-bwa run ./example-jobstore --retryCount=1 --workDir=/data --sample \
+    test-uuid file:///full/path/to/read1.fq.gz file:///full/path/to/read2.fq.gz`
 
-This pipeline produces a BAMFILE for a given sample.
+## Example Config
 
-## Running / Help
 
-It is recommended to use the associated launch scripts which provide default arguments needed to run the pipeline. 
-It is likely that the job store positional argument, `--workDir`, and `--output-dir` arguments will need to be modified.
-To run a pipeline after dependencies have been installed, simply:
+    # BWA Alignment Pipeline configuration file
```

Comment:
I indented it because a ``` block wasn't working — or at least not formatting correctly in pycharm.  I think the line of hashes screwed something up.

![screen shot 2016-06-29 at 10 51 45 am](https://cloud.githubusercontent.com/assets/7576256/16462766/97f9ee4a-3de7-11e6-8a4d-52c63af31ee4.png)